beacon-chain/core/blocks/attester_slashing.go

```diff
@@ -40,15 +41,16 @@ func ProcessAttesterSlashings(
 	beaconState state.BeaconState,
 	slashings []ethpb.AttSlashing,
 	slashFunc slashValidatorFunc,
-) (state.BeaconState, error) {
+	exitInfo *validators.ExitInfo,
+) (state.BeaconState, *validators.ExitInfo, error) {
```

Comment:
I decided to return the exit info from all functions because otherwise it's not at all obvious that the object can be modified inside the function.